compositions/aws-provider/iam-policy/velero-read.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: read-velero.iampolicy.awsblueprints.io
```

Comment:
I think we can consolidate IAM policy to S3 read and S3 write policies just for simplicity, instead of creating policies specific to applications.